app/views/Export.js

```diff
@@ -60,50 +61,54 @@ function ExportScreen({ shareButtonDisabled }) {
   async function onShare() {
     try {
       let locationData = await new LocationData().getLocationData();
-
-      const jsonData = base64.encode(JSON.stringify(locationData));
+      var options = {};
+      var jsonData = JSON.stringify(locationData);
       const title = 'PrivateKit_.json';
-      const filename = 'PrivacyKit_.json';
+      const filename = 'PrivateKit_.json';
```

Comment:
Let's make this a unique filename -- it makes the file management easier for a Redactor who is saving them all in the same folder.  This can be a simple Unix Timestamp.